docs/pages/versions/unversioned/sdk/webbrowser.md

```diff
@@ -36,6 +36,15 @@ Opens the url with Safari in a modal on iOS using `SFSafariViewController`, and
 #### Arguments
 
 - **url (_string_)** -- The url to open in the web browser.
+- **arguments (_object_)** --
+  Optional. A dictionaty with following key-value pairs:
+
+  - `toolbarColor` : a string describing a color of toolbar to be used by Custom Tab on Android. Required format `#AARRGGBB` or `#RRGGBB`. This is supported only on Android. Optional.
```

Comment:
See for example this https://github.com/expo/expo/blob/8864b1cd7ce15f5f9e058360d6474e2d02d455fd/docs/pages/versions/unversioned/sdk/notifications.md#arguments-2 and follow convention in marking `optional` and `platform only` things 😄